data.js

```diff
@@ -1,4 +1,22 @@
 module.exports = [
+  {
+    name: "2minsDevsNotes",
+    description: "Github-based fast-to-read blog about dev's daily routine challenges",
+    url: "https://github.com/vtcaregorodtcev/2minsDevsNotes",
```

Comment:
Please add your blog link here

Reply:
this is it ;)

Reply:
this is github-based blog

Reply:
@naresh-io you can visit it and see :)

Reply:
@markodenic can you please look into this?

Reply:
also, there is an article, about why writing articles on github is a very convenient way :) https://github.com/vtcaregorodtcev/2minsDevsNotes/issues/4

Reply:
I appreciate the reference link, but I'm unsure if GitHub based blogs are currently allowed. @markodenic will review it.

Reply:
@naresh-io @markodenic any updates so far?